package-lock.json

```
@@ -8568,9 +8568,9 @@
       }
     },
     "serialize-javascript": {
-      "version": "2.1.2",
-      "resolved": "https://registry.npmjs.org/serialize-javascript/-/serialize-javascript-2.1.2.tgz",
-      "integrity": "sha512-rs9OggEUF0V4jUSecXazOYsLfu7OGK2qIn3c7IPBiffz32XniEp/TX9Xmc9LQfK2nQ2QKHvZ2oygKUGU0lG4jQ==",
+      "version": "4.0.0",
```

```suggestion
      "version": ">4.0.0",
```

In order to get future upgrades?

---

No that syntax is not correct, you can't say the version that we are currently using is `4.0.0` and above.

---

I see, thanks.